src/components/CalendarDay.jsx

```diff
@@ -64,7 +76,7 @@ export default class CalendarDay extends React.Component {
         onMouseLeave={e => this.onDayMouseLeave(day, e)}
         onClick={e => this.onDayClick(day, e)}
       >
-        {day.format('D')}
+        {this.renderDay(day)}
```

Comment:
an alternative would be to get rid of the instance method entirely, and just define the defaultProp value for `renderDay` to be `day => day.format('D')`, and do `{renderDay(day)}` inline here?